concepts/05 UI Components/DataGrid/20 Editing/45 Disable Editing/40 Cell.md

```diff
@@ -0,0 +1,108 @@
+Handle the DataGrid [onEditorPreparing](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onEditorPreparing) event to conditionally set the **editorOptions.disabled** option to `true`. You also need to check the **parentType** parameter to determine a row type this cell belongs to: "dataRow" or "filterRow". To see for which editor this event raises, inspect the **dataField** parameter. Finally, check the **row.data** object that contains row data values.
```

Comment:
Why is the **editorOptions.disabled* without a link?

Reply:
Ok